### PR TITLE
Add missing px in border radius

### PR DESCRIPTION
**Pull request by coderkd10**
Fixes bug #24

### CODE REVIEW

File: src/spinners/ScaleLoader.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Loader extends React.Component {
         width: ${this.props.width}px;
         height: ${this.props.height}px;
         margin: ${this.props.margin};
```
Comment (davidhu2000) [inline src/spinners/ScaleLoader.jsx]: looks like we need to add `px` here as well.

Comment (coderkd10) [reply src/spinners/ScaleLoader.jsx]: No. margin is as a string (see L 41 below). Also the default value is already set to `2px` and this is mentioned in the documentation, so adding `px` here is not necessary.

Comment (davidhu2000) [reply src/spinners/ScaleLoader.jsx]: 👍 Sorry missed that.

**Comment by coderkd10** [comment]
@davidhu2000 thanks for approving the PR. Could you please merge it also.

**Pull request merged**